hoodie-client/src/test/java/com/uber/hoodie/table/TestMergeOnReadTable.java

```diff
@@ -326,8 +326,8 @@ public void testSimpleInsertUpdateAndDelete() throws Exception {
 
     List<String> dataFiles = roView.getLatestDataFiles().map(hf -> hf.getPath()).collect(Collectors.toList());
     List<GenericRecord> recordsRead = HoodieMergeOnReadTestUtils.getRecordsUsingInputFormat(dataFiles, basePath);
-    //Wrote 40 records and deleted 20 records, so remaining 40-20 = 20
-    assertEquals("Must contain 20 records", 20, recordsRead.size());
+    //Wrote 20 records and deleted 20 records, so remaining 20-20 = 0
+    assertEquals("Must contain 20 records", 0, recordsRead.size());
```

Comment:
fix the assert message also?  `Must contain 0 records` ?

Reply:
fixed